main.go

```diff
@@ -161,16 +163,20 @@ func main() {
 		setupLog.Error(err, "failed to create discovery client")
 		os.Exit(1)
 	}
-
+	webhookServer := webhook.NewServer(webhook.Options{
+		Port: 9443,
+	})
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		Cache:                  cache.Options{DefaultNamespaces: map[string]cache.Config{namespace: {}}},
+		WebhookServer:          webhookServer,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "2fc71baf.toolchain.member.operator",
-		Namespace:              namespace,
-		ClientDisableCacheFor:  []client.Object{&kmetrics.NodeMetrics{}},
+		Client:                 client.Options{Cache: &client.CacheOptions{DisableFor: []client.Object{&kmetrics.NodeMetrics{}}}},
```

Comment:
why do we need to exclude the `kmetrics.NodeMetrics` objects?

Reply:
I went back in the history of commits and found where it was added: https://github.com/codeready-toolchain/member-operator/commit/38d239f6ea55325ce8c947255279c5e6a4a43252
Maybe worth adding a comment